commcare_connect/connect_id_client/main.py

```diff
@@ -89,6 +90,10 @@ def get_user_otp(phone_number):
     except httpx.HTTPStatusError as e:
         if e.response.status_code == 404:
             return None
+        sentry_sdk.capture_exception(e)
+        raise
+    except httpx.HTTPError as e:
+        sentry_sdk.capture_exception(e)
```

Comment:
if we are raising the error and not handling it, it will be logged to sentry automatically

Reply:
I really don't know why I thought that sentry would not capture these exceptions